lib/index.js

```diff
@@ -1,17 +1,17 @@
 var Promise = require('bluebird');
 var dgram = require('dgram');
 
-function Sitemon(type, hosts) {
-    this._type = type;
+function Sitemon(table, hosts) {
+    this._table = table;
     this._hosts = Array.isArray(hosts) ? hosts : [hosts];
     this._totalHosts = this._hosts.length;
     this._currentHostIndex = 0;
     this._client = dgram.createSocket("udp4");
 }
 
-Sitemon.prototype.send = function(data) {
+Sitemon.prototype.send = function(keys, values) {
```

Comment:
The argument names `keys` and `values` seem to be misleading. I'm not sure what the variable names should be, but it appears that one is for "user data" and the other is for "meta data" (data about the data).